shop/src/main/java/ro/msg/learning/shop/controller/CustomerController.java

```diff
@@ -0,0 +1,54 @@
+package ro.msg.learning.shop.controller;
+
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.model.Customer;
+import ro.msg.learning.shop.payroll.CustomerNotFoundException;
+import ro.msg.learning.shop.repository.ICustomerRepository;
+
+import java.util.List;
+
+@RestController
+public class CustomerController {
+    private final ICustomerRepository repository;
+
+    CustomerController(ICustomerRepository repository){
```

Comment:
Do you need a special injection here? If not use the @RequiredArgsConstructor like you did in the previous controller